index.js

```diff
@@ -8,10 +8,10 @@ module.exports = {
   name: 'ember-normalize',
 
   treeForStyles: function() {
-    var normalizePath = path.join(this.project.nodeModulesPath, 'normalize.css');
-    var normalizeTree = new Funnel(this.treeGenerator(normalizePath), {
-      srcDir: '/',
-      destDir: '/app/styles'
+    var normalizeModulePath = path.dirname(this.project.resolveSync('normalize.css'));
```

Comment:
@webark `resolveSync` is a private method on project. `require.resolve` should work, no?

Reply:
probably.. i just started doing an import statement and dropped this addon, which should be the preferred method going forward.

Reply:
@webark I just did the same :D